tutorials/simulation_exporting.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.            super().prepare_simulation()
</pre><p>Again, I really don't like this - the code makes no sense on its own terms (super has no such method), and is prone to misuse.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>